source/ocsp-support/ocsp-support.rst

```diff
@@ -3,14 +3,14 @@ OCSP Support
 ============
 
 :Spec Title: OCSP Support
-:Spec Version: 1.0.2
+:Spec Version: 1.3.0
```

Comment:
Should this be 1.1 instead of 1.3? The previous version was 1.0.

Reply:
Oops sorry, 1.2 may actually be the correct version, since this PR (https://github.com/mongodb/specifications/pull/734) seeks to bump the spec to 1.1.